telethon/network/mtprotosender.py

```diff
@@ -315,6 +315,7 @@ async def _disconnect(self, error=None):
             for state in self._pending_state.values():
                 if error and not state.future.done():
                     state.future.set_exception(error)
+                    state.future.exception()
```

Comment:
This is silencing the exception. The user should be the one handling it, not us.

Reply:
@Lonami See other two comments.